instance/models/mixins/utilities.py

```diff
@@ -147,11 +148,30 @@ def send_urgent_alert_on_permanent_deployment_failure(sender, **kwargs):
     """
     instance = kwargs['instance']
     appserver = kwargs['appserver']
+    deployment_id: int = kwargs['deployment_id']
+
+    # Those deployment types which are triggered by OpenCraft members
+    ignorable_deployment_types: List[str] = [
+        DeploymentType.admin.name,
+        DeploymentType.batch.name,
+        DeploymentType.pr.name
+    ]
 
     # Only sending critical alerts for failures in registered clients' instances, not in test/sandboxes
     if not instance.betatestapplication_set.exists():
         return
 
+    # In case a deployment initiated by an OpenCraft member
+    # and failed, skip sending urgent email
+    if deployment_id is not None:
+        deployment = Deployment.objects.get(pk=deployment_id)
+        if deployment.type in ignorable_deployment_types:
+            logger.warning(
+                "Skip sending urgent alert e-mail after instance %s didn't provision initiated by OpenCraft member",
```

Comment:
nit: Maybe change the msg text to "Skip sending urgent alert e-mail after instance %s provisioning failed since it was initiated by OpenCraft member".

Reply:
Resolved by 76541d7